### PR TITLE
add google search scrape

### PR DESCRIPTION
**Pull request by gabrieltorresgamez**
Scrape added. Please don't close this request, i want to add the elasticsearch api.

You can use this as a guide to create your scrapers.

### CODE REVIEW

**Comment by gabrieltorresgamez** [comment]
ElasticAPI retrieval and insetion implemented.

Two questions

- [ ] Is the current env variable handling a good solution? Maybe i should add this to the make commands?
- [ ] Sometimes when scraping google, it gets stuck on the second page for 60s. Not huge problem but not perfect nonetheless...

**Comment by gabrieltorresgamez** [comment]
Feedback Fernando
- Bewertungsraster genauer analysieren
- Überall, wo eine Exception auftauchen kann -> abfangen
- Mehr Kommentare für die bessere Navigation durch den Code
- Brauchen wir beautifulsoup? besser 1 library weniger
- CSS Klassen können sich ändern -> nicht hardcodieren

**Comment by gabrieltorresgamez** [comment]
Fernando Feedback implemented.

Should I add logging and unit tests?

**Comment by gabrieltorresgamez** [comment]
how tf did the test pass with selenium????? anyway, logging, tests and typing added pls review

File: src/CryptoFraudDetection/scraper/google_results.py
Comment (can020202) [inline src/CryptoFraudDetection/scraper/google_results.py:1]: Add more functions in scraper class

**Review by can020202 — CHANGES_REQUESTED** [review]
Added comments for changes

File: src/CryptoFraudDetection/elasticsearch/__init__.py
[the remark this answers is not in the document]
Comment (gabrieltorresgamez) [reply src/CryptoFraudDetection/elasticsearch/__init__.py:1]: indeed

**Comment by gabrieltorresgamez** [comment]
implemented feedback

**Review by can020202 — APPROVED** [review]
Looks good

**Pull request merged**